.github/workflows/release.yml

```diff
@@ -16,7 +16,7 @@ on:
 # Upgrade to gcc-11 to prevent it from using its builtins (#14150)
 jobs:
   linux:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
Using a pinned version was intentional to provide broader compatibility of the builds. If you don't want to do that anymore, then remove the comment on line 15.

Reply:
I see, that makes sense. So I'll just bump those to 20.04 (the oldest supported -- by GH -- version) instead of `latest`.